### ISSUE TITLE
[TKC-1002] Disable executors during installation

### ISSUE THREAD

**Issue by jmorante-ks** [kubeshop/testkube#4720]
**Description**

Customers could require to install Testkube but without some executors. We need to implement a new feature to allow to select executors to be included in the installation.

**DoD**

* New feature to exclude executors from Testkube installation
  * For instance, add a flag in executors.json or have a few executors.json.
* Feature and documentation on how to use it.

<sub>From [SyncLinear.com](https://synclinear.com) | [TKC-1002](https://linear.app/kubeshop/issue/TKC-1002/disable-executors-during-installation)</sub>

**Comment by vsukhin** [comment]
@jmorante-ks is anything else required for this?

**Comment by jmorante-ks** [comment]
No, sorry, I do not know why it is reopened. I suppose it was caused by Linear integration that is not working properly.

If it is released and documented, no additional work required. You can close it.
